src/python/pants/backend/python/providers/pyenv/rules.py

```diff
@@ -111,6 +121,8 @@ async def get_pyenv_install_info(
         Get(EnvironmentVars, EnvironmentVarsRequest(["PATH"])),
         Get(DownloadedExternalTool, ExternalToolRequest, pyenv_subsystem.get_request(platform)),
     )
+    frozen_env_vars = FrozenDict(pyenv_subsystem.installation_env_vars)
```

Comment:
Should this use `EnvironmentVarsRequest`, in order to support the pass through syntax from `[test].extra_env_vars`, etc?

Reply:
Oh I should probably do the list-of-strings then, shouldn't I? Dang